config.py

```diff
@@ -42,6 +42,18 @@ class Config:
 
     VCAP_SERVICES = None
 
+    DM_FRAMEWORK_TO_ES_INDEX_MAPPING = {
+        'g-cloud-9': {
+            'services': 'g-cloud-9'
+        },
+        'digital-outcomes-and-specialists': {
+            'briefs': 'briefs-digital-outcomes-and-specialists-2'
```

Comment:
Please do correct me as I have very little idea about the difference between the search API mapping and the search API indexes but in terms of longevity, what is the plan for when DOS3 comes along - assuming nothing changes with how we display briefs on the brief listing page. Will we then add a new mapping for `briefs-digital-outcomes-and-specialists-3` and swap everything to it or are we intending to keep everything on the same mapping (in which case a more generic name like `briefs-digital-outcomes-and-specialists` would need no changes).

Reply:
It really depends on what happens with DOS3. If nothing much changes then it may be possible to reuse the mapping. If briefs do change, it may mean that they need a different mapping which would require a new index. If we can reuse the mapping, it's a pretty minor change to update the index name if that's what we decide to do.

Reply:
If we were to end up with something non-compatible in the data structure of DOS3 briefs, we might potentially have to define some custom process to transform the old structure into the new structure, as part of indexing. Potentially the Search API would be in charge of doing that (somehow!). It's likely very desirable to be able to continue to get all the briefs into one index, at least all the DOS ones.

Reply:
Oh, but yes probably the name of the index should just be `digital-outcomes-and-specialists` - the 'briefs' part will generally in the URLs anyway as it's the mapping type (of which as Chris said, there can only be one).

Reply:
I think the idea behind the index name, is that it's a pattern we can re-use if we have multiple objects indexed for the same framework. So if dos had services we could have `services-digital-outcomes-and-specialists-2` as well as `briefs-digital-outcomes-and-specialists-2`

Reply:
Yeah, fair enough, but I don't propose we change the g-cloud one, and it seems fairly unlikely we'll be indexing DOS services any time soon. But as it will contain DOS1 stuff, we'd still better drop the `-2`?

Reply:
I had a chat with @risicle about the `-2`. The mapping for the index is generated from the dos2 set up in the frameworks repo. If we drop the `-2` it could look like it's referring to dos1, as it's not actually called `digital-outcomes-and-specialists-1`, rather just `digital-outcomes-and-specialists`.

Reply:
Hmm, I think if the index contains both, then it would be weird for it's name to just mention one of them. Same goes for the name of the mapping, we can discuss Wednesday.

Reply:
Have had a chat with Rob, and he has no strong opinion about it, and I'm happy to change. `digital-outcomes-and-specialists` it is!